content/pizza-instructions/contents.lr

```diff
@@ -6,27 +6,34 @@ body:
 
 ### Schedule the Order with Ian's
 
-* Order 6 pizzas from Ian's Pizza State Street location, specifically requesting that they be sliced into 12 slices per pie -- 608-257-9248, ext. 1
-* Keep the order simple & vegetarian-friendly:
-  * 2 Mac&Cheese pizzas,
-  * 2 "All The Veg" pizzas, and
-  * 2 "Smokey the Bandit" pizzas
-* Ask for plates & napkins for 40
-* Please include a tip to the kitchen/driver in your payment
-* Schedule delivery to the Central Library room 302 for 6:30pm on the night of the event
-* Give them Ed's name & number as the primary contact, and he'll take the call from the delivery driver, forward the receipt info to you, etc.
+* Get in touch with a MadPy organizer who will be present for the event (via [Slack](madpy.slack.com) is preferred) to confirm your sponsorship and get their name and phone number. They will be primary contact for the order and manage the delivery at the event.
+* Order from the Ian's Pizza State Street location. Call `608-257-9248` and select option `1` at the initial prompt.
+* Order `6` `20-inch` pizzas. Keep the order simple and mostly-vegetarian-friendly. We suggest:
+  * `2` `Mac & Cheese` pizzas,
+  * `2` `All The Veg` pizzas, and
+  * `2` `Smokey the Bandit` pizzas
+* Ask for each pizza to be sliced into 12 pieces.
+* Ask for plates & napkins for 40.
+* Please include a tip in your payment. We suggest `20%`.
+* Schedule delivery to the [Madison Public Library - Central at 201 West Mifflin Street](https://maps.app.goo.gl/A4Ptcib2YM384ZcA6), room `302`, for `6:30pm`, on the night of the event.
+* Give them the MadPy organizer's name & number as the primary contact, and they will take the call from the delivery driver, forward the receipt info to you, etc.
 
 ### Provide a Blurb for Social Media
 
-We'll mention you or your organization every time we promote the event (on Slack, LinkedIn, etc). Let us know what to say! Nothing too long, since it needs to fit in a single post. If you're hiring python folks, don't hesitate to share.
+We'll mention you or your organization every time we promote the event (on Slack, LinkedIn, etc).
+Let us know what to say! 
+Nothing too long, since it needs to fit in a single post.
+If you're hiring, don't hesitate to share.
```

Comment:
Though the whitespace here makes the source easier to read, I want to note that the markdown won't render the single-spacing the way you might expect. To get the rendered output to look like this, the easiest thing to do is end each single-spaced line with two blank spaces before the line-break.

Reply:
Oh yeah, my intent was just to make the source easier to read - I do this with all my md :)